mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -621,6 +621,15 @@ def ROCDL_DPPUpdateOp : ROCDL_IntrOp<"update.dpp", [], [0],
   Arguments<(ins LLVM_Type:$old, LLVM_Type:$src, I32Attr:$dppCtrl, I32Attr:$rowMask,
       I32Attr:$bankMask, I1Attr:$boundCtrl)> {
   let results = (outs LLVM_Type:$res);
+  let summary = "Represent the DPP(Data-Parallel Primitives) functionality";
+  let description = [{
+    Represent the DPP(Data-Parallel Primitives) functionality,
```

Comment:
Typically don't repeat what you have said in `summary`. Just give more explanation. Something like "This op maps to DPP (data parallel primitive) instrinsics. It alows threads in the same subgroup to exchange data in their registers directly, following very specific exchange patterns. [and then explain in detail about the semantics..]"